.github/workflows/ci.yaml

```diff
@@ -106,7 +106,7 @@ jobs:
         working-directory: ${{ github.workspace }}/src/github.com/${{ github.repository }}
       - name: Integration tests
         run: |
-          go test -count 1 -timeout 1h -race -v ./...
+          go test -count 1 -timeout 1h -race -v -run "[a-zA-Z]*Single[a-zA-Z]*"
```

Comment:
```suggestion
          go test -count 1 -timeout 1h -race -v -run Signle
```

Reply:
It's not working without regular expression